calib:
  cam1_cam2:
    matching:
      match_th_dist: 500.0
      normal_radius: 50.0
      extract_objets: true
      kp_corr_rej: true
    transform_estimation:
      use_points: true
      use_planes: true
      compute_rotation: true
      compute_translation: true
      project_points: true
      planes_weight: 100.0
    merging:
      enable_cutting: true
      y_pose: 40.0
      width: 500.0
    tf_filtering:
      cam1_cam2:
        rejection: false
        average: true
        filter_weight: 2.0
  scene_icp:
    tf_filtering:
      rejection: false
      average: true
      filter_weight: 2.0
  scene:
    icp:
      enable: true
      subsize: 40.0
      autom: false
      max_dist: 500.0
      max_it: 500
      epsilon: 5.0